pytorch_toolkit/formula_recognition/tools/utils/exporter.py

```diff
@@ -0,0 +1,147 @@
+                              0: 'batch', 1: 'H', 2: 'W'}}
+                          )
+
+    def export_encoder_ir(self):
+        input_model = self.config.get("res_encoder_name")
+        input_shape = self.config.get("input_shape_decoder")
+        output_names = self.config.get("encoder_output_names", ENCODER_OUTPUTS)
+        export_command = f"""{OPENVINO_DIR}/bin/setupvars.sh && \
+        python {OPENVINO_DIR}/deployment_tools/model_optimizer/mo.py \
+        --framework onnx \
+        --input_model {input_model} \
+        --input_shape "{input_shape}" \
+        --output "{output_names}" \
+        --reverse_input_channels \
+        --scale_values 'imgs[255,255,255]'"""
+        if self.config.get('verbose_export'):
+            print(export_command)
+        subprocess.run(export_command,
+                       shell=True, check=True
+                       )
+
+    def export_decoder_ir(self):
+        input_shape_decoder = self.config.get("input_shape_decoder")
+        output_h, output_w = input_shape_decoder[2] / 32, input_shape_decoder[3] / 32
+        if self.config['backbone_config']['disable_layer_4']:
+            output_h, output_w = output_h * 2, output_w * 2
+        if self.config['backbone_config']['disable_layer_3']:
+            output_h, output_w = output_h * 2, output_w * 2
+        output_h, output_w = math.ceil(output_h), math.ceil(output_w)
+        input_shape = [[1, self.config.get('head', {}).get("decoder_hidden_size", 512)],
+                       [1, self.config.get('head', {}).get('decoder_hidden_size', 512)],
+                       [1, self.config.get('head', {}).get('encoder_hidden_size', 256)],
+                       [1, output_h, output_w, self.config.get('head', {}).get('decoder_hidden_size', 512)], [1, 1]]
+        input_shape = "{}, {}, {}, {}, {}".format(*input_shape)
+        input_model = self.config.get('res_decoder_name')
+        input_names = self.config.get("decoder_input_names", DECODER_INPUTS)
+        output_names = self.config.get("decoder_output_names", DECODER_OUTPUTS)
+        export_command = f"""{OPENVINO_DIR}/bin/setupvars.sh &&
+        python {OPENVINO_DIR}/deployment_tools/model_optimizer/mo.py \
+        --framework onnx \
+        --input_model {input_model} \
+        --input {input_names} \
+        --input_shape '{input_shape}' \
+        --output {output_names}"""
+        if self.config.get('verbose_export'):
+            print(export_command)
+        subprocess.run(export_command,
+                       shell=True, check=True
+                       )
+
+    def export_model_if_not_yet(self, model, model_type, ir=False):
+        assert model_type in ('encoder', 'decoder')
+        result_model_exists = os.path.exists(model)
+        if ir:
+            model_xml = model.replace(".onnx", '.xml')
+            model_bin = model.replace(".onnx", '.bin')
+            result_model_exists = os.path.exists(model_xml) and os.path.exists(model_bin)
```

Comment:
```suggestion
        if ir:
            model_xml = model.replace(".onnx", '.xml')
            model_bin = model.replace(".onnx", '.bin')
            result_model_exists = os.path.exists(model_xml) and os.path.exists(model_bin)
            if not result_model_exists:
                # export to ONNX if it is required
                self.export_model_if_not_yet(model, model_type, ir=False)
        else:
            result_model_exists = os.path.exists(model)
```
This will make the method safe w.r.t. absence of ONNX models for OpenVINO export